src/Entity/EntityCustomField.php

```diff
@@ -159,4 +164,20 @@ public static function getStorageFieldName()
     {
         return 'entityValue';
     }
+
+    public function __clone()
+    {
+        $this->id = null;
+        $data = $this->getEntityData();
+        if (is_array($data)) {
+            $newData = new ArrayCollection();
```

Comment:
simpler:
`$newData = new ArrayCollection($data->toArray()); `

Reply:
slightly more advanced mechanism required since $data can be array or ArrayCollection.

Reply:
I see, forget about it.